pkg/cloud/services/eks/securitygroup.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func (s *Service) reconcileSecurityGroups(cluster *eks.Cluster) error {
-	s.scope.Info("Reconciling EKS security groups", "cluster-name", cluster.Name)
+	s.scope.Info("Reconciling EKS security groups", "cluster-name", *cluster.Name)
```

Comment:
Please use `pointer.String` in case for any reason that name is left empty.

Reply:
`pointer.String` gets the reference, `pointer.StringDeref` dereferences the string, but requires a default value which would be empty. Ill update to that, but Im not sure that it changes much

Reply:
Yah that's fine.